packages/@aws-cdk/aws-gamelift-alpha/lib/build.ts

```diff
@@ -276,4 +313,13 @@ export class Build extends BuildBase {
     });
   }
 
+  private validateServerSdkVersion(serverSdkVersion?: string) {
+    if (!serverSdkVersion) return;
+    if (!serverSdkVersion.match(/^\d+\.\d+\.\d+$/)) {
+      throw new Error(`serverSdkVersion must be in the 0.0.0 format, got ${serverSdkVersion}.`);
+    }
+    if (serverSdkVersion.length > 128) {
+      throw new Error(`serverSdkVersion length must be smaller than or equal to 128, got ${serverSdkVersion}.`);
```

Comment:
I think it would be better to output the length of a character in the error message. I have seen other cases like this.

https://github.com/aws/aws-cdk/blob/main/packages/aws-cdk-lib/aws-stepfunctions-tasks/lib/sagemaker/create-training-job.ts#L341
https://github.com/aws/aws-cdk/blob/main/packages/aws-cdk-lib/aws-events-targets/lib/batch.ts#L118
https://github.com/aws/aws-cdk/blob/main/packages/%40aws-cdk/aws-glue-alpha/lib/database.ts#L136


```suggestion
      throw new Error(`serverSdkVersion length must be smaller than or equal to 128, got ${serverSdkVersion.length}.`);
```